tools/mydumper.md

```diff
@@ -0,0 +1,46 @@
+---
+title: mydumper Instructions 
+summary: Use mydumper to export data from TiDB.
+category: tools
+---
+
+# mydumper Instructions
+
+## What is mydumper?
+
+`mydumper` is a fork of the [mydumper](https://github.com/maxbube/mydumper) project with additional functionality specific to TiDB. It is the recommended method to use for logical backups of TiDB.
+
+[Download the Binary](http://download.pingcap.org/tidb-enterprise-tools-latest-linux-amd64.tar.gz).
+
+## What enhancements does this contain over regular mydumper?
+
++ Use `tidb_snapshot` to provide backup consistency instead of `FLUSH TABLES WITH READ LOCK`
+
++ Include the hidden `_tidb_rowid` column in exports when present
+
++ Allow for `tidb_snapshot` to be configurable (i.e. backup data as it appeared at an earlier point in time)
+
+
+### New parameter description
+
+```
+  -z, --tidb-snapshot: Set the tidb_snapshot to be used for the backup.
+                       Default: NOW()-INTERVAL 1 SECOND.
```

Comment:
This can be any datetime expression (e.g. `-z "'2018-10-10 22:22:22'"`)? Is there any link to the `tidb_snapshot` feature?

Reply:
It is documented here: https://pingcap.com/docs/op-guide/history-read/#how-tidb-reads-data-from-history-versions

Reply:
I've tried to make it clear.  links can't be used in code blocks, so I've added a link above when describing the feature.